vxquery-core/src/test/java/org/apache/vxquery/datamodel/ObjectByteTest.java

```diff
@@ -252,4 +253,10 @@ public void testItemNotExistObject() {
             Assert.fail("Test failed to write the object pointable.");
         }
     }
+
+    private boolean compareKeys(IPointable tvp1, IPointable tvp2) {
+
+        return FunctionHelper.arraysEqual(tvp1.getByteArray(), tvp1.getStartOffset() + 1, tvp1.getLength() - 1,
+                tvp2.getByteArray(), tvp2.getStartOffset(), tvp2.getLength());
+    }
```

Comment:
I am using this method because the pointable returned from getKeys has tag and the key I have created don't have tag. This is to compare both. Why use getValue method of TaggedValuePointable in this case? Sorry about the confusion. It is ok to have +1 and -1 in a test method. Isn't it?